tee-worker/ts-tests/integration-tests/identity.test.ts

```diff
@@ -214,14 +247,15 @@ describeLitentry('Test Identity', 0, (context) => {
             },
         };
         const bobSubject = await buildIdentityFromKeypair(new PolkadotSigner(context.substrateWallet.bob), context);
-
+        const nonce9 = await getNonce(base58mrEnclave, workerAddress, context);
```

Comment:
It's strange that nonce`9` is used here and later again `nonce` is named. Can we use `const nonce` here?